frontend/src/metabase/components/EntityMenu.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import type { ComponentStory } from "@storybook/react";
+import EntityMenu from "./EntityMenu";
+
+export default {
+  title: "Entity Menu",
```

Comment:
We should put this in a folder, since we currently have no top-level stories. I have no idea where it should go. Probably `Components/`

```suggestion
  title: "Components/Entity Menu",
```